applications/stative_network.md

```diff
@@ -0,0 +1,312 @@
+|    0c. | Testing and Testing Guide                       | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests.                                                                                                                                                                                                                                                                                                                                                           |
+|    0d. | Docker                                          | We will provide a Dockerfile that can be used to test all the functionality delivered with this milestone.                                                                                                                                                                                                                                                                                                                                                                                                                |
+|    0e. | Article                                         | We will publish an **article** on our bloq showcasing the achievements created in this grand application. Further, this includes snippets from the documentation to give interested reader a good starting point.                                                                                                                                                                                                                                                                                                         |
+|     1. | Substrate module: NST                           | Implement the NST substrate module (assets, balances, tx, fees) uses Quote module pricings (MS1.2). This includes the stablecoin calculation logic (blueprint for all FIAT tracking NSTs). The final deliverable here is a Substrate module (incl. updated `src`, Dockerfile, documentation & tutorial).                                                                                                                                                                                                                  |
+|     2. | Substrate smart contract (Ink): NST-Swap-Brdige | Implement the NST bridge smart contract for the Stative Network using Substrates contract module w/ _ink!_. This bridge contract will in the beginning only work with USDT withdrawals. Further stable coins will be added in the future. Allows to bridge-swap an NST from the Stative Netwotk to USDT on ETH. Allows to bridge an NST from the Stative Netwotk to a wrapped version on other Blockchains. The final deliverable here is a Substrate module (incl. updated `src`, Dockerfile, documentation & tutorial). |
```

Comment:
Do you want to develop your own bridge here, or does this leverage an existing solution? ([Snowbridge](https://github.com/Snowfork/snowbridge) for example?)

Reply:
Yes, here we are planning to develop our own bridges. Main reason is the fact, that we - again - have specific requirements due to the concept of aggregated sets backing the NSTs and governance token. Our bridges 
a. need to be able to lock different tokens as part of the aggregated sets that are backing the NSTs and
b. need to be able to lock NSTs (so a native token of the stative parachain) as part of an aggregated set that is backing the governance token
For bridges to enable minting wrapped tokens for the native tokens of our stative parachain, we will definitely check leveraging existing solutions already working with ETH and the like.
Last but not least due to the recent developments and incidents in the context of bridges, we feel the need to build a solution from zero with max focus on security and a very thorough auditing.

Reply:
In this case, could you provide more information about your bridge? I personally see a huge risk in setting up your own independent chain rather than simply using a parachain slot, at least in the beginning. Implementing a decentralized bridge is a very difficult task.